tests/plugins/test_add_labels.py

```diff
@@ -176,6 +180,12 @@ def test_add_labels_plugin(tmpdir, docker_tasker,
     flexmock(workflow, base_image_inspect=labels_conf_base)
     setattr(workflow.builder, 'df_path', df.dockerfile_path)
 
+    if reactor_config_map:
+        workflow.plugin_workspace[ReactorConfigPlugin.key] = {}
+        workflow.plugin_workspace[ReactorConfigPlugin.key][WORKSPACE_CONF_KEY] =\
+            mocked_reactorconfig({'version': 1, 'image_labels': deepcopy(labels_conf),
+                                  'image_equal_labels': eq_conf})
+
```

Comment:
all instances of this code should be replaced by a common function that takes optional arguments for image_labels and image_equal_labels and sets up the reactor config map, like the function in test_fetch_maven_artifacts

Reply:
ok